src/framework/filterControl/filterDropdown/CreateFilterFormDropdown.jsx

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
`FilterDropdownForm` would be a better name because it is a form that is rendered on top of the `FilterDropdowns`.